pos_customer_wallet_partner_is_user/static/src/js/screens.js

```diff
@@ -0,0 +1,38 @@
+odoo.define("pos_customer_wallet_partner_is_user.screens", function (require) {
+    "use strict";
+    var core = require("web.core");
+    var screens = require("point_of_sale.screens");
+
+    var _t = core._t;
+
+    screens.PaymentScreenWidget.include({
+        /**
+         * Overload function.
+         *
+         * - If client hasn't enabled functionality, don't allow wallet payments.
+         */
+        order_is_valid: function () {
```

Comment:
the [original function](https://github.com/coopiteasy/OCB/blob/12.0/addons/point_of_sale/static/src/js/screens.js#L2081) has a `force_validation` argument, which is missing here, but also [in `pos_customer_wallet`](https://github.com/coopiteasy/addons/blob/12.0/pos_customer_wallet/static/src/js/screens.js#L26). the argument should be passed through to the the `_super()` call in both modules. a way to do this without caring for existing arguments is:

```javascript
this._super.apply(this, arguments);
```